torch/autograd/profiler.py

```diff
@@ -430,6 +461,7 @@ def _privateuse1_memory_usage(mem_record):
 
             cpu_memory_usage = 0
             cuda_memory_usage = 0
+            xpu_memory_usage = 0
```

Comment:
I think we rename "cuda_memory_usage" to "gpu_memory_usage". Then just call _gpu_memory_usage() and let that handle the mem_record. currently cuda_memory_usage works for both NVIDIA CUDA GPUs and AMD GPUs

Reply:
I am just trying not to modify _cuda named code. It should be the next step to consolidate and standardize the naming conventions and to maximize the reuse of GPU-related variables and logic as much as possible.